test/compilable/testdip1008.d

```diff
@@ -0,0 +1,20 @@
+// PERMUTE_ARGS:
```

Comment:
Missing argument to actually test the dip.

Reply:
done

Reply:
but it'll never work until the druntime changes are pulled, so I disabled it

Reply:
That's actually a good thing, it will prevent this P.R. from being pulled before it's ready.

Reply:
what @mathias-lang-sociomantic said!

Reply:
Which came first, the chicken or the egg?